fem/fe.hpp

```diff
@@ -294,6 +294,15 @@ class FiniteElement
           CURL  ///< Implements CalcCurlShape methods
         };
 
+   /** @brief Enumeration for ContType: defines the continuity of the
+       field across element interfaces.
+   */
+   enum { CONTINUOUS,   ///< Field is continuous across element interfaces
+          TANGENTIAL,   ///< Tangential components of vector field
+          NORMAL,       ///< Normal component of vector field
+          DISCONTINUOUS ///< Field is discontinuous across element interfaces
+        };
```

Comment:
Nice!

Reply:
This should be a property of the FE collection (that's where the inter-element connectivity is described), not the `FiniteElement`. For example, you can use any `FiniteElement` to create a discontinuous collection, e.g. using class `Local_FECollection`.

Reply:
OK, I don't actually use it here.  I have similar methods in the `FE_Collection` which are actually being used.  I can move the `enum` to `FiniteElementCollection` and remove the relevant methods from `fe.hpp`.